packages/react-native-sdk/docusaurus/docs/reactnative/05-ui-cookbook/13-hosting-a-livestream.mdx

```diff
@@ -0,0 +1,98 @@
+---
+id: hosting-a-livestream
+title: Hosting a livestream
+---
+
+This cookbook tutorial walks you through how to customize the UI for hosting a livestream on React Native SDK.
+
+### Default component
+
+Our React Native SDK provides a specialized components for it: the [`HostLivestream`](../../ui-components/livestream/host-livestream).
```

Comment:
@khushal87 Probably line 8 can replace line 12 here? What do you think?

Reply:
Yeah, makes sense

Reply:
Haha, I'm sorry for the miscommunication. 

Specifically, I meant, under Default Component, instead of starting the section with "Our React Native SDK provides a specialized components for it: the [`HostLivestream`](../../ui-components/livestream/host-livestream).", you can directly replace it with "For the host role, our React Native SDK includes the specialized `HostLivestream` component.".

I hope this makes it clear!

Reply:
Ah okay. I have fixed it. Thanks 😄